packages/docusaurus-plugin-content-docs/src/index.ts

```diff
@@ -120,18 +120,18 @@ export default function pluginContentDocs(
       Object.keys(docsMetadataRaw).forEach(currentID => {
         let previous;
         let next;
-        const previousID = idx(docsMetadataRaw, [currentID, 'previous']);
+        const previousID = docsMetadataRaw[currentID]?.previous;
         if (previousID) {
           previous = {
-            title: idx(docsMetadataRaw, [previousID, 'title']) || 'Previous',
-            permalink: idx(docsMetadataRaw, [previousID, 'permalink']),
+            title: docsMetadataRaw[previousID]?.title ?? 'Previous',
```

Comment:
I guess i need to revert this. Babel preset TS and prettier cant parse optional chaining yet